README.md

```diff
@@ -21,6 +21,8 @@ You can find the latest Spark documentation, including a programming
 guide, on the [project web page](https://spark.apache.org/documentation.html).
 This README file only contains basic setup instructions.
 
+### Instructions for installing Apache Spark on Linux [See here](https://github.com/drakelam/Apache-Spark-Tutorial/blob/main/Apache-Spark-Tutorial/Apache-Spark-Installation.md "See here")
```

Comment:
This is already described below `Building Spark` (https://spark.apache.org/docs/latest/building-spark.html). Feel free to improve the docs there.

Reply:
Thanks, I will try harder!